lib/backpex/html/resource/resource_index_table.html.heex

```diff
@@ -102,8 +113,8 @@
         ]}
       >
         <div
-          class="flex items-center justify-end space-x-2"
-          x-bind:style="`height: ${rowHeight - 1}px !important; width: ${actionsWidth - 32}px !important`"
```

Comment:
> @makl9797 What is the status of this PR? Please mark as draft if applicable. Please request a review when ready.

I have removed the highlighted part so that the width of the column is `auto`. Previously the width was fixed, which meant that `setActionsWidth` did not work correctly either, as the width that is set in `setActionsWidth` is based on the previously set width. The problem now is that the column is no longer displayed correctly when the page is loaded due to the missing initial width setting. The correct width is calculated and set for a short time, but then immediately overwritten again. I have not yet been able to find out why this is the case, but the problem could be solved with a delay of half a second. However, this is not a reasonable solution to the problem. I have not worked on it any further so far.